src/components/Modal/BaseModal.tsx

```diff
@@ -110,13 +98,10 @@ function BaseModal({
         if (shouldSetModalVisibility) {
             Modal.setModalVisibility(true);
         }
-        onModalShow();
+        onModalShow?.();
     };
 
-    const handleBackdropPress = (e) => {
-        if (e && e.key === CONST.KEYBOARD_SHORTCUTS.ENTER.shortcutKey) {
```

Comment:
We we are removing that?

Reply:
It looks to me like this event handler doesn't accept any parameters

Reply:
@JKobrynski Please test this extensively and confirm if the event is being sent or not (maybe in some platforms is being sent and others not) in order to avoid regressions

Reply:
Turns out event is actually present but only on web, I brought it back!